load_sample_data.sh

```diff
@@ -17,31 +17,40 @@ docker-compose exec -T "$WEB_SERVICE_NAME" /bin/bash -c "python3 manage.py shell
+	EOF"
+
+# Load sample data for images
+docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "PGPASSWORD=deploy pg_dump -s -t image -U deploy -d openledger -h db | psql -U deploy -d openledger"
 docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "psql -U deploy -d openledger <<-EOF
 	ALTER TABLE image RENAME TO image_view;
 	ALTER TABLE image_view ADD COLUMN standardized_popularity double precision;
-	CREATE TABLE content_provider(provider_identifier varchar(50), provider_name varchar(250), created_on timestamp, domain_name varchar(500), filter_content boolean, notes text, media_type varchar(80));
-	INSERT INTO content_provider (created_on, provider_identifier, provider_name, domain_name, filter_content) VALUES (now(), 'flickr', 'Flickr', 'https://www.flickr.com', false, 'image'), (now(), 'behance', 'Behance', 'https://www.behance.net', false, 'image');
 	\copy image_view (id,created_on,updated_on,identifier,provider,source,foreign_identifier,foreign_landing_url,url,thumbnail,width,height,filesize,license,license_version,creator,creator_url,title,tags_list,last_synced_with_source,removed_from_source,meta_data,tags,watermarked,view_count,standardized_popularity) from './sample_data/sample_data.csv' with (FORMAT csv, HEADER true)
 	EOF"
 
-# This is currently added for testing purposes, we can adjust once the audio support is added
-#docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "psql -U deploy -d openledger <<-EOF
-#	CREATE TABLE IF NOT EXISTS audio_view (id serial,identifier uuid,created_on timestamp,updated_on timestamp,ingestion_type varchar(1000),provider varchar(1000),source varchar(1000),foreign_identifier varchar(1000),foreign_landing_url varchar(1000),url varchar(1000),thumbnail varchar(1000),duration integer,bit_rate integer,sample_rate integer,category varchar(1000),genre jsonb,audio_set jsonb,alt_audio_files jsonb,filesize integer,license varchar(1000),license_version varchar(25),creator varchar(1000),creator_url varchar(1000),title varchar(1000),meta_data jsonb,tags jsonb,last_synced_with_source timestamp,removed_from_source boolean,standardized_popularity double precision);
-#	\copy audio_view (identifier,created_on,updated_on,ingestion_type,provider,source,foreign_identifier,foreign_landing_url,url,thumbnail,duration,bit_rate,sample_rate,category,genre,audio_set,alt_audio_files,filesize,license,license_version,creator,creator_url,title,meta_data,tags,last_synced_with_source,removed_from_source,standardized_popularity) from './sample_data/sample_audio_data.csv' with (FORMAT csv, HEADER true)
-#	EOF"
+# Load sample data for audio
+docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "PGPASSWORD=deploy pg_dump -s -t audio -U deploy -d openledger -h db | head -n -14 | psql -U deploy -d openledger"
```

Comment:
What does this line do? Could we remove the extra step of altering tables (image -> image_view, audio -> audio_view) and create the necessary tables from the start?

Reply:
This creates the `audio_view` table in the upstream DB as a replica of the `audio` table from the API DB. The pipe to `head` (`| head -n -14 `) is for removing the `audio`&rarr;`audio_set` FK.

The benefit is that it prevents us from having to declare all the fields manually and just use the autogenerated DB schema from Django's models.

Reply:
Thank you for clear explanations!